main/CheckBox/index.tsx

```diff
@@ -1,20 +1,105 @@
-import React, { FC } from 'react';
+import React, { FC, useCallback, useState } from 'react';
 import {
-  Text,
-  View,
+  TouchableHighlight,
 } from 'react-native';
 
-interface Props {
-  defaultChecked?: boolean,
-  disabled?: boolean,
+import styled from 'styled-components/native';
+
+type MarkerContainerProps = {
+  boxSize?: number
+  boxColor?: string
+}
+
+type LabelProps = {
```

Comment:
Need to change to use `Interface`.

Reply:
> The name of the folder called `CheckBox` should also be changed to `Checkbox`. But sometimes case change could not work with git as our expectation. Be careful.

through the link below, I was able to resolve the issue of git not recognizing this case change.
- https://stackoverflow.com/questions/17683458/how-do-i-commit-case-sensitive-only-filename-changes-in-git
- https://git-scm.com/docs/git-config#Documentation/git-config.txt-coreignoreCase